demos/notebooks/dqx_demo_library.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DQX - Use as library demo\n",
+    "\n",
+    "In this demo we demonstrate how to create and apply a set of rules from an object and from a YAML configuration. \n",
+    "\n",
+    "**Note.**\n",
+    "This notebook can be executed without any modifications when using the `VS Code Databricks Extension`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Instal DQX"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install databricks-labs-dqx==0.5.0\n",
```

Comment:
Please don't pin version. The demo must be runnable with the latest version. We don't want to maintain this manually. We need to add execution of demos as part of the testing suite to make sure they are runnable. I added separate ticket for this.